Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '~> 3.1.0'
+ruby '~> 3.2.2'
```

Comment:
Maybe we should not require an exact version here, but use `~> 3.2`?

Reply:
`~> 3.2.2` is already not an exact version. It means `>= 3.2.2 && < 3.3`, which seems like the specifity we want for Ruby versions, since the minor version number is a new language level that requires more care.

Reply:
ah, yes. nvm